sources/packages/backend/apps/api/src/route-controllers/student-appeal/_tests_/e2e/student-appeal.aest.controller.getAppeals.e2e-spec.ts

```diff
@@ -0,0 +1,83 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import { DataSource } from "typeorm";
+import {
+  AESTGroups,
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  getAESTToken,
+} from "../../../../testHelpers";
+import {
+  E2EDataSources,
+  createE2EDataSources,
+  createFakeStudentAppeal,
+  createFakeStudentAppealRequest,
+  saveFakeApplicationDisbursements,
+  saveFakeStudent,
+} from "@sims/test-utils";
+import { ApplicationStatus, StudentAppealStatus } from "@sims/sims-db";
+import { getUserFullName } from "../../../../utilities";
+
+describe("StudentAppealAESTController(e2e)-getAppeals", () => {
+  let app: INestApplication;
+  let appDataSource: DataSource;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    appDataSource = dataSource;
+    db = createE2EDataSources(dataSource);
+  });
+  it("Should return only one row in the ministry appeals dashboard when a one searched for a student application that has an appeal.", async () => {
+    // Arrange
+    // Create student to submit application.
+    const student = await saveFakeStudent(appDataSource);
+    // Create application to request change.
+    const application = await saveFakeApplicationDisbursements(
+      db.dataSource,
+      {
+        student,
+      },
+      { applicationStatus: ApplicationStatus.Completed },
+    );
+    // Create approved student appeal.
+    const appealRequest = createFakeStudentAppealRequest(
+      {},
+      { initialValues: { appealStatus: StudentAppealStatus.Pending } },
+    );
```

Comment:
For the sake of ensuring the list is returned properly with multiple appeal requests, please make sure that at least 2 appeal requests are present for the appeal.